hugo/content/exporter/_index.md

```diff
@@ -96,6 +95,18 @@ The following pgmonitor configuration files should be placed according to the fo
 | postgres/queries_backrest.yml | `/etc/postgres_exporter/##/queries_backrest.yml` |
 | postgres/pgbackrest-info.sh | `/usr/bin/pgbackrest-info.sh` |
 
+### Windows Packages
+
+The following Windows Server 2012R2 packages are available to [Crunchy Data](https://www.crunchydata.com) customers through the [Crunchy Customer Portal](https://access.crunchydata.com/). *After installing using the below packages, continue reading at the [Windows Server 2012R2](#in-server-2012R2) section.*
```

Comment:
"After installing using the below packages" -> "After installing below package" ?

Reply:
After installing via